kitsune/wiki/strategies.py

```diff
@@ -93,9 +97,15 @@ def translate(self, l10n_request: TranslationRequest) -> TranslationResult:
         return TranslationResult(success=True, method=TranslationMethod.MANUAL, metadata={})
 
 
+@dataclass
 class AITranslationStrategy(TranslationStrategy):
     """AI-based translation strategy without human review."""
 
+    draft_manager: "AIContentManager" = field(init=False)
```

Comment:
Why `draft_manager`? It handles much more than drafts, so it seems this should be `content_manager` instead.